src/data/fintype.lean

```diff
@@ -449,6 +449,23 @@ instance finset.fintype [fintype α] : fintype (finset α) :=
 instance subtype.fintype [fintype α] (p : α → Prop) [decidable_pred p] : fintype {x // p x} :=
 set_fintype _
 
+instance psigma.fintype {α : Type*} {β : α → Type*} [fintype α] [∀ a, fintype (β a)] : 
+  fintype (Σ' a, β a) := 
+fintype.of_equiv _ (equiv.psigma_equiv_sigma _).symm
+
+instance psigma.fintype_prop_left {α : Prop} {β : α → Type*} [∀ a, fintype (β a)] [decidable α] : 
```

Comment:
Is there a reason why you put the dependent type class assumption before the assumption on \alpha? In the preceding declaration you didn't, but in these three you did...

Reply:
didn't really think about it.

Reply:
I have thought about it now. Rob's rule is to put the thing that fails faster first, and I'm guessing that's more likely to be fintype, but I don't really know.